src/compas_fab/robots/robot.py

```diff
@@ -548,7 +609,27 @@ def get_group_names_from_link_name(self, link_name):
         return group_names
 
     def get_position_by_joint_name(self, configuration, joint_name, group=None):
-        """Returns the value of the joint_name in the passed configuration.
+        """Returns the joint value of the joint with ``joint_name`` in the passed configuration.
+
+        Parameters
+        ----------
+        configuration : :class:`Configuration`
+            The configuration of the configurable joints.
+        joint_name : str
+            Name of joint.
+        group : str, optional
+            The name of the planning group. By default the main planning group.
+
+        Returns
+        -------
+        list of floats
```

Comment:
```suggestion
        float
```
Returns only one float iirc.